company.py

```diff
@@ -0,0 +1,111 @@
+10. Вывести названия должностей, которые получают больше 90к без повторений.
+11. Посчитать среднюю зарплату по каждому отделу среди девушек (их зовут Мишель, Николь, Кристина и Кейтлин).
+12. Вывести без повторений имена людей, чьи фамилии заканчиваются на гласную букву.
+Третий уровень:
+Теперь вам пригодится ещё список taxes, в котором хранится информация о налогах на сотрудников из разных департаметов.
+Если department None, значит, этот налог применяется ко всем сотрудникам компании.
+Иначе он применяется только к сотрудникам департмента, название которого совпадает с тем, что записано по ключу department.
+К одному сотруднику может применяться несколько налогов.
+13. Вывести список отделов со средним налогом на сотрудников этого отдела.
+14. Вывести список всех сотредников с указанием зарплаты "на руки" и зарплаты с учётом налогов.
+15. Вывести список отделов, отсортированный по месячной налоговой нагрузки.
+16. Вывести всех сотрудников, за которых компания платит больше 100к налогов в год.
+17. Вывести имя и фамилию сотрудника, за которого компания платит меньше всего налогов.
+"""
+
+departments = [
+    {
+        "title": "HR department",
+        "employers": [
+            {"first_name": "Daniel", "last_name": "Berger", "position": "Junior HR", "salary_rub": 50000},
+            {"first_name": "Michelle", "last_name": "Frey", "position": "Middle HR", "salary_rub": 75000},
+            {"first_name": "Kevin", "last_name": "Jimenez", "position": "Middle HR", "salary_rub": 70000},
+            {"first_name": "Nicole", "last_name": "Riley", "position": "HRD", "salary_rub": 120000},
+        ]
+    },
+    {
+        "title": "IT department",
+        "employers": [
+            {"first_name": "Christina", "last_name": "Walker", "position": "Python dev", "salary_rub": 80000},
+            {"first_name": "Michelle", "last_name": "Gilbert", "position": "JS dev", "salary_rub": 85000},
+            {"first_name": "Caitlin", "last_name": "Bradley", "position": "Teamlead", "salary_rub": 950000},
+            {"first_name": "Brian", "last_name": "Hartman", "position": "CTO", "salary_rub": 130000},
+        ]
+    },
+]
+
+taxes = [
+    {"department": None, "name": "vat", "value_percents": 13},
+    {"department": "IT Department", "name": "hiring", "value_percents": 6},
+    {"department": "BizDev Department", "name": "sales", "value_percents": 20},
+]
+
+def money_position(departments):
+    dep_money = []
+    for employers in departments:
+        title = employers['title']
+        money = 0
+        for employers_info in employers['employers']:
+            money += employers_info['salary_rub']
+        dep_money.append(f'{title}: {money}')
+    return dep_money
+
+def position(departments):
+    job_title = []
+    for employers in departments:
+        for employers_info in employers['employers']:
+            if employers_info['salary_rub'] < 80000:
+                job_title.append(employers_info['position'])
```

Comment:
если сразу сделать job_title множеством
```suggestion
                job_title.add(employers_info['position'])
```